css/properties.schema.json

```diff
@@ -169,6 +169,7 @@
         "sameAsMinWidthAndMinHeight",
         "sameAsWidthAndHeight",
         "specifiedIntegerOrAbsoluteLength",
+        "specifiedKeyword",
```

Comment:
this is perhaps not needed since we have:

https://github.com/mdn/data/blob/2e6ee0843a895ca7d71485b4899a53f8746ec430/css/properties.schema.json#L177